### PR TITLE
Room daniyar

### PR DESCRIPTION
**Pull request by DaniyarS**

### CODE REVIEW

File: app/src/main/java/com/example/groupproject/AuthorizationFragment.kt [hunk 1]
```diff
@@ -2,191 +2,152 @@ package com.example.groupproject
...
+            btSign = it.findViewById(R.id.btRegistrate)
             accountFragment = AccountFragment()
-            progressBar = it.findViewById(R.id.progressBar2)
-            nMainNav = it.findViewById(R.id.main_nav)
             progressBar2.hide()
         }
-
-        signIn.setOnClickListener() {
-            initNewToken()
+        btSign.setOnClickListener(){
```
Comment (DaniyarS) [inline app/src/main/java/com/example/groupproject/AuthorizationFragment.kt:71]: We added space between ){ before pull. It's magic

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 2]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
-    @SerializedName("poster_path") val poster_path: String?=null,
-    @SerializedName("backdrop_path") val backdrop_path: String?=null,
-    @SerializedName("title") val title: String?=null,
-    @SerializedName("overview") val overview: String?=null,
-    @SerializedName("release_date") val release_date: String?=null,
-    @SerializedName("runtime") val runtime: Int = 0
+    val idMovie: Int?=null,
+    @SerializedName("id") val id: Int? = null,
+    @SerializedName("popularity") val popularity: Double? = null,
+    @SerializedName("vote_count") val vote_count: Int? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:15]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 3]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
 data class Movie(
     @PrimaryKey
-    @SerializedName("id") val id: Int?=null,
-    @SerializedName("poster_path") val poster_path: String?=null,
-    @SerializedName("backdrop_path") val backdrop_path: String?=null,
-    @SerializedName("title") val title: String?=null,
-    @SerializedName("overview") val overview: String?=null,
-    @SerializedName("release_date") val release_date: String?=null,
-    @SerializedName("runtime") val runtime: Int = 0
+    val idMovie: Int?=null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:12]: add spaces.

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 4]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("id") val id: Int? = null,
+    @SerializedName("popularity") val popularity: Double? = null,
+    @SerializedName("vote_count") val vote_count: Int? = null,
+    @SerializedName("video") val video: Boolean? = null,
+    @SerializedName("poster_path") val poster_path: String? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:17]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 5]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("adult") val adult: Boolean? = null,
+    @SerializedName("backdrop_path") val backdrop_path: String? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:19]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 6]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("original_language") val original_language: String? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:20]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 7]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("original_title") val original_title: String? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:21]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 8]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("selected") var selected: Int? = 0,
+    @SerializedName("title") val title: String? = null,
+    @SerializedName("vote_average") val vote_average: Double? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:24]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 9]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
+    @SerializedName("overview") val overview: String? = null,
+    @SerializedName("release_date") val release_date: String? = null,
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:26]: use camelCase

File: app/src/main/java/com/example/groupproject/model/Movie.kt [hunk 10]
```diff
@@ -4,49 +4,33 @@ import androidx.room.Entity
...
-//    @SerializedName("original_title") val original_title: String,
-//    @SerializedName("genre_ids") val genre_ids: List<Int>,
-//    @SerializedName("genres") val genres: List<MovieGenres>,
-//    @SerializedName("vote_average") val vote_average: Double,
-//    @SerializedName("results") val results: List<Movie>
-)
+) : Serializable
+{
+    fun getPosterPathImage(): String {
+        return "https://image.tmdb.org/t/p/w342"+poster_path }
```
Comment (Mura75) [inline app/src/main/java/com/example/groupproject/model/Movie.kt:32]: use camelCase and add spaces

**Pull request merged**